### ISSUE TITLE
Convertor keeps both files around?

### ISSUE THREAD

**Issue by cchance27** [aszc-dev/ComfyUI-CoreMLSuite#16]
I noticed the convertor is keeping the mlpackage and models around after conversion, shouldn't the convertor drop the prelude file to save space (maybe an option in the node to keep the uncompiled version, as with each size needing a model, having 2x the files adds up quick.

Technically we end up with 3x models, the modelc the mlpackage and the safe tensor, would be nice to make keeping that middle one optional, and by default to just remove it after compilation.

**Comment by aszc-dev** [comment]
I was thinking the same thing. Indeed, I don't see any purpose for mlpackage file in our use-case atm. For now, I think it's safe to delete that file. I don't want to bloat the interface with additional option (it seems a bit crowded as it is). I will change that in some future update.

**Comment by cchance27** [comment]
ya definitly feels like cleaning up the intermediate is safe

**Comment by x4080** [comment]
Hi, i just doing the conversion from realistic vision with target size 512x768,original,cpu and gpu, the result is only .mlpackage

but maybe because of error :
```
xcrun: error: unable to find utility "coremlcompiler", not a developer tool or in PATH
ERROR:root:!!! Exception during processing !!!
```

is something not installed in my mac ?

Thanks

**Comment by x4080** [comment]
Ok after I investigate further the command 'xcrun coremlcompiler compile' that makes that errors. How to install "coremlcompiler" ?

**Comment by cchance27** [comment]
did you have xcode installed when you installed the plugin?

**Comment by x4080** [comment]
@cchance27 Yes I did, so I was surprised why its not working. Hope it can help someone

**Comment by aszc-dev** [comment]
@x4080 
I don't know if you're still facing this issue, but if you are, there's an easy solution described in [this article](https://mybyways.com/blog/fast-stable-diffusion-using-core-ml-on-m1) (It's in the last paragraph of Model Conversion section). Perhaps this could help?

**Comment by x4080** [comment]
@aszc-dev already solved it thanks, it was xcode problem
